modules/extensions-spring-support/src/test/resources/models/test-fn.json

```diff
@@ -275,8 +275,8 @@
               "expressionSupport": "SUPPORTED",
               "role": "BEHAVIOUR",
               "dslConfiguration": {
-                "allowsInlineDefinition": true,
-                "allowsReferences": true,
+                "allowsInlineDefinition": false,
```

Comment:
why?

Reply:
IdempotentDeclarationWalker now walks thorugh function parameters, enrichers that use this walker now takes them into account . IMO this values for allowsInlineDefinition and allowsReferences makes for sense than the ones before